lib/blackbox/VmBlackBox.rb

```diff
@@ -217,479 +151,58 @@ def validate(settings, check_vmId = true)
         end
       end
 
-			return {:results=>{:created=>false, :exists=>self.exist?, :configured=>self.configured?, :bbName=>@bbName, :vmName=>@config_name},
-				      :config=>@cfg}
+      return {:results=>{:created=>false, :exists=>self.exist?, :bbName=>@bbName, :vmName=>@config_name},
+              :config=>@cfg}

```

Comment:
@Fryguy I think that entire method can be removed as it is used in calls when creating/deleting the blackbox files which are no longer used.  While looking into this I may have found some additional dead code related to this that I will try to remove.  Will mark as WIP for now until I can get back to it.